frontend/src/collapsible.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * sri sri guru gaurangau jayatah
+ */
+
+window.collapsible = () => {
+  return {
+    expanded: false,
+
+    toggle() {
+      this.expanded = !this.expanded;
+    },
+    itemsList: {
+      'x-show'() {
+        return this.expanded;
+      },
+      searchHeader: {
```

Comment:
1. The `collapsible` component can be used anywhere, not necessarily in the filter panels.
2. The shorter is better. In the context of the component it is enough to say `header`.

Reply:
Moreover, for some reason this object became a child of `itemsList`. This is improper, it will not work even.